.github/workflows/ci.yml

```diff
@@ -103,6 +103,10 @@ jobs:
           if ((Get-FileHash 'sqliteodbc_w64.exe').Hash -ne "0df79be4a4412542839ebf405b20d95a7dfc803da0b0b6b0dc653d30dc82ee84") {Write "Hash mismatch"; Exit 1}
           ./sqliteodbc_w64.exe /S
         shell: powershell
+      - id: remove-strawberry-perl-patch-exe
```

Comment:
can you update the step ID, too, please?

Reply:
Of course, updated in f38c9b5e67e69027dbb67152e9c849f9408f5e61